imagersite/imager_images/templates/imager_images/library.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
+{% load thumbnail %}
+
+{% block content %}
+<p>{{albums.first.user.username}}'s Albums:</p>
```

Comment:
This will not display a username if the current user has no albums.

Reply:
done